export function handleError(error: any): void {
    return void 0;
}